### ISSUE TITLE
Adding a Data Legend can mess up the graph display.

### ISSUE THREAD

**Issue by taviso** [taviso/123elf#109]
Adding a Data Legend can mess up some graph displays. It's still legible, but doesn't look right.

**Comment by rruhle** [comment]
Yes, long graph titles, at top of screen, works much better than data legends
